### PR TITLE
Pass loaded trip to Google Maps for saved trip

### LINKED ISSUE #380: Saved Trips Sometimes Not Loading Correctly
Loading a saved trip should work as follows:
merp

As you can see, everything got populated and the app downloaded the best route and displayed it for you all on its own.

When Plan Trip is the first page to use Maps, we see the following behavior:
merp

Simply, nothing happens.  I suspect that [this line](https://github.com/umts/pvta-multiplatform/blob/master/src/pages/plan-trip/plan-trip.component.ts#L201) of code is the culprit; we're never passing `loaded trip` to Google Maps when we need to get maps **before** doing anything else.

This issue will begin to associate you with the sometimes-unwieldy Plan Trip code, how we use Google Maps, and Ionic's lifecycle events.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #380 
(hopefully I got this right. It works, but I don't know if I coded properly).

### CODE REVIEW

**Comment by akaplo** [comment]
This does solve the problem.  FWIW I realize now, however, that the problem was in fact created by my own mediocre code that I'm still unsure how to fix.

The [MapService function ](https://github.com/umts/pvta-multiplatform/blob/master/src/providers/map.service.ts#L14) that downloads Google Maps gets passed the correct Plan Trip function as a callback, but the issue is that it **doesn't** [get passed the loadedTrip param](https://github.com/umts/pvta-multiplatform/pull/403/files#diff-8f802a9b80a60c0a01b7ed67374a46b8R201).  Additionally, I don't think you can just pass parameters to the Google Maps callback that the MapService creates.

Long story short, @mboneil10 solved the problem with a good workaround and I think we should merge this as a hotfix, but the underlying issue still exists until we can figure out a better design pattern here.

**Review by akaplo — APPROVED** [review]
